AgoraSocialApp/SvelteKit/src/routes/$layout.svelte

```diff
@@ -1,30 +1,24 @@
 <script lang="ts">
-    //import { AuthService } from "$components/Auth/auth.service";
-    import { setContext } from 'svelte';
-    import Nav from '$components/Nav.svelte';
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  
+  import './global.css';
+  import { AuthService } from '$components/Auth/auth.service';
+  import Logo from '$components/UI/Logo.svelte';
+  import { onMount } from 'svelte';
+  import { user } from '$stores/user';
+  import { isAuthed, token } from '$stores/session';
 
-    //const authService = new AuthService();
-    //setContext('authService', authService);
-
-
-	//import { goto } from '$app/navigation';
-	//goto('/new/route');
+  const authService = AuthService.getInstance();
 
+  onMount(async () => {
```

Comment:
as this is a route, shouldn't it be in the `load` function?

Reply:
currently oidc isn't setup for ssr, i thought you had said there was an issue with that?

Reply:
updated